superset/db_engine_specs/base.py

```diff
@@ -387,6 +387,19 @@ def df_to_sql(cls, df: pd.DataFrame, **kwargs):  # pylint: disable=invalid-name
         """
         df.to_sql(**kwargs)
 
+    @classmethod
+    def dialect_paramstyle(cls, sqla_uri: str) -> Optional[str]:
```

Comment:
I don't agree we should be exposing overriding of the `paramstyle` (set by the dialect in the DB-API) here as it seems that it only pertains to uploading CSVs.